error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -297,6 +297,23 @@ boolean after(Stream<T> stream) {
     }
   }
 
+  /**
+   * When using {@link Map#get(Object)} in a {@link Stream#map(Function)} operation, prefer
```

Comment:
I would rephrase this to `Prefer {@link ..} over {@link ..} to avoid double look-up` or something :).